settings.gradle

```diff
@@ -145,8 +149,8 @@ include ":sdks:python:test-suites:tox:py2"
 include ":sdks:python:test-suites:tox:py35"
 include ":sdks:python:test-suites:tox:py36"
 include ":sdks:python:test-suites:tox:py37"
-include ":vendor:grpc-1_21_0"
-include ":vendor:bytebuddy-1_9_3"
+include ":vendor:grpc-1_26_0"
+include ":vendor:bytebuddy-1_10_8"
```

Comment:
The vendored versions should not be changed in this PR. Probably better rebase this PR. Focus in the settings on changes for the PR